backend/dashboard_metrics/tasks.py

```
@@ -0,0 +1,458 @@
+                    if agg["labels"]:
+                        obj.labels = {**obj.labels, **agg["labels"]}
+                    obj.save(
+                        update_fields=[
+                            "metric_value",
+                            "metric_count",
+                            "labels",
+                            "modified_at",
+                        ]
+                    )
+                    updated += 1
+            except Exception as e:
```

Ruff (0.14.10)

Do not catch blind exception: `Exception`

(BLE001)